rules/manager.go

```diff
@@ -49,7 +45,7 @@ var (
 			Name:      "rule_evaluation_duration_seconds",
 			Help:      "The duration for a rule to execute.",
 		},
-		[]string{ruleTypeLabel},
+		[]string{"rule_type", "status"},
```

Comment:
For the record, I questioned whether it makes sense to observe latency of failures in the same metric. Failure latency will either be near 0 or near the timeout and make it harder to understand what my success latency is. @brian-brazil What do you think about this problem?